apps/analytics/urls.py

```diff
@@ -1,4 +1,10 @@
 from django.conf.urls import url  # noqa
```

Comment:
Just a small suggestion here, I think now we can remove `# noqa` from here as there will be no need to ignore this line from `flake8` now.